tesserocr.pyx

```diff
@@ -2069,6 +2069,11 @@ cdef class PyTessBaseAPI:
                 renderer = new TessOsdRenderer(outputbase)
                 return renderer
 
+        self._baseapi.GetBoolVariable("tessedit_create_alto", &b)
+        if b:
+            IF TESSERACT_VERSION >= 0x3999800:
+                renderer = new TessAltoRenderer(outputbase)
+
```

Comment:
Since this is only supported in version `>=0x3999800` doesn't it make more sense to do:

```python
IF TESSERACT_VERSION >= 0x3999800:
    self._baseapi.GetBoolVariable("tessedit_create_alto", &b)
    if b:
        renderer = new TessAltoRenderer(outputbase)
```

Reply:
I fail to see the difference. (But we could raise an exception if you prefer...)

Reply:
it just makes sense when reading the code mainly and to avoid useless operations for older versions